sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -1940,8 +2010,8 @@ CREATE TABLE `person_patient_link` (
         (`form_id`, `field_id`, `group_id`, `title`, `seq`, `data_type`, `uor`, `fld_length`, `max_length`, `list_id`, `titlecols`, `datacols`, `default_value`, `edit_options`, `description`, `fld_rows`)
     VALUES
         ('DEM','additional_telecoms',@group_id,'',@max_seq+9,55,0,0,0,'',4,4,'','[\"J\",\"SP\"]','Additional Telecoms',0);
-    #Endif
-#Endif
+    #EndIf
```

Comment:
Have we tested this to ensure nesting works (I don't think nesting works, but if it is working on your testing, then it must :) ).

Reply:
While I didn't add this one you call this out on me a couple years ago when I used and was working but I removed because I didn't research in code to be 100%.
I saw this one and paid attention on testing and seems to work.
Then again I though misspelled `#Endif` use to break run but didn't see here or happenstance just allowed success.
Since parse is sequential and an open/close tags I'd think it would work.

Reply:
I thought I removed all of David's nested ifs, on my testing they didn't work, so maybe something has changed and now they work.

Reply:
Don't go by me guys because i'm not 100% but will look at in code because I know it worked for me in the past the time I tried. Still if you want I will refactor unless you can get to @adunsulag